### DISCUSSION TITLE
Italic numbers in math mode

### DISCUSSION THREAD

**Discussion by yochem** [typst/typst#2803] [Q&A] [answered]
![image](https://github.com/typst/typst/assets/23235841/d79b3266-0c6e-41b8-b52c-ff000ef2e061)

Text on image:

Desired result #emph("1") instead of 1. In math mode:

```typst
$ italic(1) quad 1 $
```
Gives:

$ italic(1) quad 1 $

`italic("1")` does the same.

I also tried `$ text(emph("1")) $`, which gives an error.

**Comment by Enivex** [comment, marked as answer]
There are no italic numbers in unicode, so this cannot be achieved without fake italics or by using a non-math font (like you did with emph).

Numbers are always typeset upright in traditional math typesetting.
